frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts

```diff
@@ -83,6 +126,18 @@ export class BookingsPageComponent extends BaseComponent implements OnInit {
         window.open(link);
     }
 
+    private loadMeetings(meetingMemberRequest: IMeetingMembersRequest) {
+        this.meetingService
+            .getThreeMeetings(meetingMemberRequest)
```

Comment:
We need to change method name

Reply:
Changed method name to getMeetings(meetingMemberRequest).